src/components/Avatar/Avatar.tsx

```diff
@@ -50,6 +50,12 @@ export interface AvatarProps extends BaseComponentProps {
    * @default AvatarSize.MEDIUM
    */
   size?: AvatarSize;
+
+  /**
+   * Decides whether to fade the Avatar image in each render, pass through to Persona
+   * @default false
+   */
+  imageShouldStartVisible?: boolean;
```

Comment:
Should we keep the same prop name we used with `Image`? https://github.com/Microsoft/YamUI/blob/9d6a78a3a25241dfb8b38becec8b022bccdee1ca/src/components/Image/Image.tsx#L48-L52

Reply:
the interesting thing from Persona is it has these both:-

```ts
    /**
     * If true, adds the css class 'is-fadeIn' to the image.
     */
    imageShouldFadeIn?: boolean;
    /**
     * If true, the image starts as visible and is hidden on error. Otherwise, the image is hidden until
     * it is successfully loaded. This disables imageShouldFadeIn.
     * @defaultvalue false
     */
    imageShouldStartVisible?: boolean;
```

I'll change it to use `imageShouldFadeIn`

Reply:
done